rebuild.sh

```diff
@@ -116,41 +226,97 @@ mvnBuildDocker() {
     14)
       mvnImage=maven:${mvnVersion}-jdk-${jdk}
       ;;
-    11 | 15 | 16 | 17)
```

Comment:
no, please don't change existing defaults: they are used by existing buildspecs

Reply:
I agree: Defaults must remain the same.